src/components/Avatar/Styles/group.scss

```diff
@@ -7,7 +7,10 @@
 
   &.spaced {
     :not(:first-child):not(.avatar-group-tooltip) {
-      margin-left: -$space-xxxs;
+      margin-left: $space-xxxs;
+    }
+    & .avatar {
```

Comment:
`AvatarGroup` needs to support border too. This should be removed.

Reply:
Pass the border thickness from SASS to TypeScript an example may be found in `Slider` of how this is done.

Reply:
From there you may be able to account for the offset.

Reply:
Exported from SCSS:
https://github.com/EightfoldAI/octuple/blob/main/src/components/Slider/slider.module.scss#L30-L43

Typed:
https://github.com/EightfoldAI/octuple/blob/main/src/components/Slider/Slider.types.ts#L6-L32

Referenced:
https://github.com/EightfoldAI/octuple/blob/main/src/components/Slider/Slider.tsx#L220

Reply:
Reverted this change because the status item shifting was fixed by other two changes:
1. use of `transform` in placement of status items
2. added `:not(.avatar-status-item)` to `src/components/Avatar/Styles/group.scss`